package.json

```diff
@@ -46,6 +46,7 @@
     "@types/node": "^18.0.0",
     "@vitest/ui": "^0.17.0",
     "c8": "^7.11.3",
+    "esbuild": "^0.14.48",
```

Comment:
?

Reply:
The build on node 14 started failing with `npm ERR! esbuild-android-64 not accessible from esbuild`. I couldn't see any changes to dependencies different to other branches (that built successfully) or find any issue or reason for the error, but adding esbuild as a direct dependency fixed it.